portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,35 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Tyler!</h1>");
+    response.setContentType("application/json;");
+    ArrayList<String> list = new ArrayList<String>();
+    list.add("Hey this is message number 1");
+    list.add("AND this is message number 2");
+    list.add("couldn't forget about message #3");
+    String json = convertToJson(list);
+    System.out.println(json);
+    response.getWriter().println(json);
+  }
+
+   private String convertToJson(ArrayList<String> list) {
```

Comment:
Curious: what would happen if convertToJson(...) was called with an input of > 3 strings?